tools/CoqMakefile.in

```diff
@@ -510,7 +510,8 @@ vok: $(VOFILES:%.vo=%.vok)
 .PHONY: vok
 
 validate: $(VOFILES)
-	$(TIMER) $(COQCHK) $(COQCHKFLAGS) $(COQLIBS_NOML) $^
+	$(TIMER) $(COQCHK) $(COQCHKFLAGS) $(COQLIBS_NOML) $(PROFILE_ARG) $^
+	$(HIDE)$(PROFILE_ZIP)
```

Comment:
This seems wrong to me. The profile file of `$(PROFILE_ARGS)` and `$(PROFILE_ZIP)` is `$<.prof.json`, that is, the first `.vo` file from `$(VOFILES)`. Perhaps I am missing something, but it seems to me the profile file here should be `$@.prof.json`.

Reply:
changed